File: JAVA/Assignment/Constructor/Code1.java
class Demo{
	int x= 10;
	Demo(int x){
		System.out.println(" in demo constructor");
	}
	void fun(){
		System.out.println(x);
	}
	public static void main(String ag[]){
		Demo obj = new Demo(10);
		obj.fun();
	}
}

File: Python/ClassCode/Class/Code1.py
class Employee:
    print ("In Microsoft class")
    name= "Sakshi"
    empId=123
    sal=7
    CompanyName= "Microsoft"
    def work(self):
        print("Software Developer:")
        print("Name: "+self.name)
        print("empId: "+str(self.empId))
        print("Com Name: "+self.CompanyName)
obj=Employee()# call goes to constructor
obj.work()# Internally work(obj) argument was passes

